index.yaml

```diff
@@ -440,8 +482,8 @@ entries:
     - https://github.com/longhorn/longhorn-tests
     - https://github.com/longhorn/backing-image-manager
     urls:
-    - assets/longhorn/longhorn-1.1.100.tgz
-    version: 1.1.100
```

Comment:
Can we modify the version for existing entries? Similar below
Need to wait for Rancher team's review.

Reply:
This is automatically generated. 

@jiaqiluo any idea for this or what else we need to change in package.yaml?

Reply:
cc @aiyengar2

Reply:
The items are ordered from the largest version to the smallest, so we see those changes here.
the real changes are the additions of longhorn 1.2.0 and longhorn-crd 1.2.0, which is expected.